index.html

```diff
@@ -26,5 +26,183 @@ <h1>José Fernández-Caminero</h1>
+            </button>
+        </section>
+
+        <section class="box1">
+            <img src="resources/Rectangle 21(6).png" alt="Picture of laptop and other objects">
+
+            <h3 class="projectName">Project name goes here</h3>
+
+            <ul class="greenul">
+                <li class="greenli">HTML/CSS</li>
+                <li class="greenli">Ruby on Rails</li>
+                <li class="greenli">JavaScript</li>
+            </ul>
+
+            <button class="bordersolid">See this project
+                <img src="resources/Union.png" alt="Cool Arrow Pointing Right">
+            </button>
+        </section>
+    </section>
+
+    <section>
+        <h1>About</h1>
+
+        <p>
+            I love to code. I think coding is where the arts and the sciences meet. It's the most natural activity for a renaissance person.
+        </p>
+
+        <button>Get my CV
+            <img src="resources/ic_download.svg" alt="Downward Arrow">
+        </button>
+
+        <section class="marginleft">
+            <section class="rectangle">
+                <img src="resources/Rectangle 41.png" alt="Blue Rectangle">
+
+                <img class="rectanglesymbol1" src="resources/pound.svg" alt="Pound Sign">
+                
+                <section class="rectangletext">
+                    <h3 class="maryPoppins">Languages</h3>
+
+                    <ul>
+                        <li class="maryPoppins2">JavaScript(ES5/ES6)</li>
+                        
+                        <li class="maryPoppins2">Ruby</li>
+                        
+                        <li class="maryPoppins2">HTML</li>
+                        
+                        <li class="maryPoppins2">CSS</li>
+                    </ul>
+                </section>
+            </section>
+
+            <section class="rectangle">
+                <img src="resources/Rectangle 39.png" alt="Red Rectangle">
+
+                <img class="rectanglesymbol2" src="resources/Group 18.png" alt="Square inside a Square">
+                
+                <section class="rectangletext2">
+                    <h3 class="maryPoppins">Frameworks</h3>
+                    
+                    <ul>
+                        <li class="maryPoppins2">Bootstrap</li>
+                        
+                        <li class="maryPoppins2">Ruby on rails</li>
+                        
+                        <li class="maryPoppins2">RSpec</li>
+                        
+                        <li class="maryPoppins2">Capibara</li>
+                        
+                        <li class="maryPoppins2">Selenium</li>
+                    </ul>
+                </section>
+            </section>
+
+            <section class="rectangle">
+                <img src="resources/Rectangle 42.png" alt="Yellow Rectangle">
+
+                <img class="rectanglesymbol3" src="resources/ic_skills.svg" alt="Lit Lightbulb">
```

Comment:
Kindly, decoration images should be part of the background of your site, I recommend using background-image